src/DApp.js

```diff
@@ -18,16 +18,26 @@ class App extends React.Component {
       if (err) {
```

Comment:
It's not shown here, but if you could replace `connect` by `onReady`, it'd be better, because `connect` is no longer needed.

I had forgotten to change it back.